src/main/java/com/github/steveice10/mc/protocol/codec/MinecraftCodec.java

```diff
@@ -3,186 +3,28 @@
 import com.github.steveice10.mc.protocol.data.ProtocolState;
 import com.github.steveice10.mc.protocol.data.game.level.event.LevelEventType;
 import com.github.steveice10.mc.protocol.data.game.level.sound.BuiltinSound;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundCustomPayloadPacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundDisconnectPacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundKeepAlivePacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundPingPacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundResourcePackPopPacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundResourcePackPushPacket;
-import com.github.steveice10.mc.protocol.packet.common.clientbound.ClientboundUpdateTagsPacket;
-import com.github.steveice10.mc.protocol.packet.common.serverbound.ServerboundClientInformationPacket;
-import com.github.steveice10.mc.protocol.packet.common.serverbound.ServerboundCustomPayloadPacket;
-import com.github.steveice10.mc.protocol.packet.common.serverbound.ServerboundKeepAlivePacket;
-import com.github.steveice10.mc.protocol.packet.common.serverbound.ServerboundPongPacket;
-import com.github.steveice10.mc.protocol.packet.common.serverbound.ServerboundResourcePackPacket;
+import com.github.steveice10.mc.protocol.packet.common.clientbound.*;
+import com.github.steveice10.mc.protocol.packet.common.serverbound.*;
```

Comment:
No wildcard imports please, you can disable this in IDEA by setting the minimum number to 1000+

if you find existing wildcards, i would just fix them on a case-by-case basis if you're already making changes to that class.